.github/workflows/create_deployment.yml

```diff
@@ -1,10 +1,11 @@
 name: Create deployment
 on:
   push:
```

Comment:
@robintw - I presume we can reinstate the next two lines, so that only tags on the `master` branch trigger releases.

Well, once we're happy it's all working ok.

Reply:
Actually, no - as git tags aren't actually related to branches. I thought originally they were - but it seems not, see https://stackoverflow.com/questions/14613540/do-git-tags-only-apply-to-the-current-branch#:~:text=Here's%20another%20way%20of%20putting,only%20ever%20identify%20a%20commit.

That is, there is no way to find only tags from the master branch - so I think this will run on _any_ tags (but as we only use tags for releases, I think that's fine).

Reply:
Aah, this is one workaround: https://github.community/t/run-workflow-on-push-tag-on-specific-branch/17519/4

Reply:
Ok, I'll create another PR to test that later.